packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -184,9 +205,17 @@ const Card = React.forwardRef(function Card(
         elevation,
     });
 
+    const ariaLabelValue = labels?.cardAriaLabel
+        ? labels.cardAriaLabel
+        : ariaLabelledBy
+          ? undefined
+          : undefined;
+
     return (
         <View
-            aria-label={labels?.cardAriaLabel}
+            aria-busy={ariaBusy}
+            aria-label={ariaLabelValue}
+            aria-labelledby={ariaLabelledBy}
```

Comment:
Based on the prop docs: 
>  2. `aria-labelledby` - For ID references (will take precedence over `aria-label` if both are provided).

There's a case where both the `aria-label` and `aria-labelledby` attributes can be set! Does `aria-label` need to be `undefined` if `aria-labelledby` is set?